modules/infra/digitalocean/main.tf

```diff
@@ -49,6 +49,47 @@ resource "digitalocean_droplet" "droplet" {
   }
 }
 
+resource "digitalocean_loadbalancer" "k8s_api_loadbalancer" {
```

Comment:
Can you make this optional? You can check other modules for inspiration. For example, the "create_vpc" variable in AWS.

Reply:
Made these changes in https://github.com/rancher/tf-rancher-up/pull/147/commits/31ea706bcaad436b1d3be2c0e063c14f98ff39f3

Reply:
Handled this in https://github.com/rancher/tf-rancher-up/pull/147/commits/31ea706bcaad436b1d3be2c0e063c14f98ff39f3